sermant-integration-tests/database-write-prohibition-test/pom.xml

```diff
@@ -35,12 +35,6 @@
     </dependencyManagement>
 
     <profiles>
-        <profile>
-            <id>opengauss-test</id>
-            <modules>
-                <module>opengauss-demo</module>
-            </modules>
-        </profile>
```

Comment:
Keep this profile

Reply:
The OpenGauss and PostgreSQL share a demo, which does not need to be reserved.